src/platform/device.gni

```diff
@@ -117,6 +117,11 @@ declare_args() {
   }
 }
 
+declare_args() {
+  # Enable Joint Fabric features
+  chip_device_config_enable_joint_fabric = false
```

Comment:
There are no tests for the new code. Is that intended?

Reply:
Indeed, no tests at this moment. @robertfarnum will follow-up immediately in a new PR with the tests and with some refactoring of the joint commissioning method state machine. Just to ease this ownership transfer, I would like if possible to signal the possible issues from the current code and allow us to introduce the tests in a follow up PR. Pls note that all the newly introduced code is under `CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC` and in order to introduce some tests there is a dependency on https://github.com/project-chip/connectedhomeip/pull/34637.

Reply:
Resolving this conversation (also announced Tennessee on Slack).